airbyte-integrations/connectors/source-file/source_file/spec.json

```diff
@@ -148,8 +148,8 @@
                 "type": "string"
               },
               "port": {
-                "type": "number",
-                "default": 22
+                "type": "string",
```

Comment:
could you try:
https://github.com/airbytehq/airbyte/blob/4c8ce600a7041c3677266dec734bc5715ea42402/airbyte-integrations/connectors/source-mysql/src/main/resources/spec.json#L15-L23

Reply:
try to use **integer** type.

Reply:
I tried this as well 😅 neither yielded a non-string value